streamnative/README.md

```diff
@@ -0,0 +1,81 @@
+# StreamNative
+
+## Overview
+
+[StreamNative][1] provides an enterprise-grade messaging and event streaming platform built on Apache Pulsar. It offers scalable, real-time data streaming solutions with features like multi-tenancy, geo-replication, and seamless integration with cloud services.
+
+The StreamNative integration collects below types of metrics:
```

Comment:
Can we please get a link to the types of metrics collected?

Reply:
Done.